app/src/main/java/uk/co/philpotter/masteremu/FileBrowser.java

```diff
@@ -913,6 +913,14 @@ public void onClick(DialogInterface dialog, int which) {
             }
         }
     }
+  public static String zerosToDate(int time, int length) {
```

Comment:
Please sort the indentation on this function, and put an empty line between it and the previous one.